Graylog2/graylog2-server
# Adjust resolution of dashboard layout for wide screens

## Description
Double the resolution of the dashboard layout grid. And add a migration
to adjust the existing dashboard widgets positions and settings. 

## Motivation and Context
If you have a big screen one column takes 1/6 of the screen. If you have
a static number widget than 1/6 is a huge widget just for showing one number.

## How Has This Been Tested?
- Add a dashboard with a couple of widgets and place them so their positions get stored to the database
- Apply migration and load new code by restarting the server
- Go to the dashboard page and the widgets position and sizes shouldn't have changed
- Ensure that the amount of columns have doubled
- Restart server to ensure that the migration did not apply a second time

## Types of changes


## Checklist:

Fixes #4306
